auth/src/include/firebase/auth.h

```diff
@@ -502,6 +502,25 @@ class Auth {
   /// Gets the App this auth object is connected to.
   App& app();
 
+  /// @brief Modifies this Auth instance to use the specified keychain access
+  /// group.
+  ///
+  /// Accessing the keychain requires that the application has the keychain
```

Comment:
Please remove all of this and instead tell the developer to refer to the Firebase iOS SDK documentation.